### PR TITLE
Fix ThrottlingException

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Fix ThrottlingException

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
s3 integration tests have been temporary disabled and a corresponding issue has been created https://github.com/ansible-collections/amazon.cloud/issues/42

File: tests/integration/targets/s3/aliases [hunk 1]
```diff
@@ -1,2 +1,4 @@
 cloud/aws
 zuul/aws/cloud_control
+
+disabled  # Temporary disabled because they throw several ThrottlingExceptions and it needs further investigation
```
Comment (jillr) [inline tests/integration/targets/s3/aliases:4]: Can you please make sure we have a ticket to remind us to come back to this?

**Review by jillr — APPROVED** [review]

File: tests/integration/targets/s3/aliases (hunk 1 above)
Comment (alinabuzachis) [reply tests/integration/targets/s3/aliases:4]: I created an issue here https://github.com/ansible-collections/amazon.cloud/issues/42. We do not have the jira label in this repo. If necessary, I can manually create a Jira ticket as well.

**Pull request merged**